integration/kubernetes/k8s-footloose.bats

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+issue="https://github.com/kata-containers/runtime/issues/1674"
+
+setup() {
+	skip "test not working see: ${issue}"
```

Comment:
Nit: I'd consider adding a blank line after each `skip` to make it clearer that all functions are being skipped currently.